<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Handoff</class>
 <widget class="QDialog" name="Handoff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QComboBox" name="emailBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonAdd">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTextEdit" name="textUsers">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="handoffConfirm">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonHandoffCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
